packages/palette/src/elements/ColumnGrid/ColumnGrid.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useMemo } from "react"
+import styled from "styled-components"
+import { gridColumn, GridColumnProps } from "styled-system"
+import { Box, BoxProps } from "../Box"
+import { CSSGrid, CSSGridProps } from "../CSSGrid"
+
+/** ColumnGrid implements `Box` and the common grid properties */
+export type ColumnGridProps = CSSGridProps
+
+/**
+ * A 12-column fluid grid. Collapses to a single column at mobile breakpoints.
```

Comment:
In the event that I want to some day override `gridColumn` to break out of the single mobile column, it might be hard for me to figure out how to do that. I'm not sure if this is the right place to put it, but is there a docs comment we can put somewhere that would tell me that setting `gridColumn` explicitly will let me break out of the single mobile column?

Reply:
Taking @damassi's suggestion simplified this a lot actually, so now there really isn't a reason to use gridColumn directly.

Included an example in the kitchen sink story.
![](http://static.damonzucconi.com/_capture/mMLYwUsT3WWq.gif)